<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWidget</class>
 <widget class="QWidget" name="ListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ListWidget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>361</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="messagepage">
       <widget class="QScrollArea" name="newslist">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>359</width>
          <height>509</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>357</width>
           <height>507</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="friendspage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="userLayout"/>
  </widget>
  <widget class="QPushButton" name="addFri">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>698</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mode">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>698</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>360</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="toolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>690</y>
     <width>360</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="messagepushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>消 息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="friendspushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>联 系 人</string>
   </property>
  </widget>
  <zorder>friendspushButton</zorder>
  <zorder>toolBar</zorder>
  <zorder>userBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>addFri</zorder>
  <zorder>mode</zorder>
  <zorder>messagepushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
